### DISCUSSION TITLE
How to disable chakra-ui-color-mode

### DISCUSSION THREAD

**Discussion by kumaraguru1735** [reflex-dev/reflex#2428] [General]
As I am using my own custom design like background color , own button text color, I do not want to use the default one
And I want to make own color-mode for cakra ui!
Please guide me Thanks!

**Comment by picklelo** [comment]
We are migrating soon to radix components, and will have a builtin `rx.color` module that you can override with your own color schemes. This will be coming in the next couple of months, then users can define their own custom color schemes / palettes.

**Comment by kumaraguru1735** [comment]
How to Set the default chakra-ui-color-mode, when startup ?
